modules/issue/template/unmarshal.go

```diff
@@ -95,14 +95,20 @@ func unmarshal(filename string, content []byte) (*api.IssueTemplate, error) {
 	}{}
 
 	if typ := it.Type(); typ == api.IssueTemplateTypeMarkdown {
-		templateBody, err := markdown.ExtractMetadata(string(content), it)
-		if err != nil {
-			return nil, err
-		}
-		it.Content = templateBody
-		if it.About == "" {
-			if _, err := markdown.ExtractMetadata(string(content), compatibleTemplate); err == nil && compatibleTemplate.About != "" {
-				it.About = compatibleTemplate.About
+		if templateBody, err := markdown.ExtractMetadata(string(content), it); err != nil {
+			// can't extract metadata, so it could be a pure markdown.
```

Comment:
It doesn't look good .... what if there are errors in the yaml inside markdown?

Reply:
Good question, but I think it could be very complicated. Think about a markdown template like this:

```
---
# Welcome!
---

What did you find?
```

It actually means two horizontal lines here. 😂

Reply:
Then a comment describing these cases should be enough? Then future developers can understand these cases and know that it's done by purpose.

Reply:
Done.